src/Garage.jsx

```diff
@@ -2,30 +2,35 @@ import { GenerateCar } from "./Helpers/CarGenerate"
 import './App.scss';
 import { getGarage } from "./Redux/ActionsCreators";
 import axios from "axios";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { useDispatch } from "react-redux/es/exports";
+import { Form } from "./Components/Form";
+import { SingleCar } from "./Components/SingleCar";
 export const Garage = ({cars})=>{
```

Comment:
The component is start to be big. A function should do only one thing. If you code start to do couple thing (in that case the component display list of car, add a new car, update cars and etc.). If you face with those situation you should separate logic to separate function/component hook. It will improve readability of your code, simplify the logic (because it easier to understand small pieces of code) and simplify any changes in feature